### PR TITLE
[Improvement-4751][API] Improvement the duration field modify the display to a time-based minute-second format (e.g. 1d 10h 20m 1s)

### LINKED ISSUE #4751: [Improvement][API] Improvement the duration field modify the display to a time-based minute-second format (e.g. 1d 10h 20m 1s)
**Describe the question**
The duration field of workflow instance and task instance page shows how many seconds to run, which is not intuitive enough. Modify the display to a time-based minute-second format (e.g. 1d 10h 20m 1s).

**Which version of DolphinScheduler:**
 -[dev]

image

### PR DESCRIPTION
**Pull request by zhuangchong**
## What is the purpose of the pull request
this PR closes #4751 
The duration field of workflow instance and task instance page shows how many seconds to run, which is not intuitive enough. Modify the display to a time-based minute-second format (e.g. 1d 10h 20m 1s).

## Verify this pull request
This change added tests and can be verified as follows:
  - *Manually verified the change by testing locally.*

image

### CODE REVIEW

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/ProcessInstance.java [hunk 1]
```diff
@@ -202,7 +202,7 @@
      * @return
      */
     @TableField(exist = false)
-    private Long duration;
+    private String duration;
```
Comment (CalvinKirs) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/ProcessInstance.java:205]: hi, should we separate this and create different beans?
It is not recommended to use one for DO, DTO, and VO.

**Comment by CalvinKirs** [comment]
looks good to me, the point mentioned in my comment may need to be discussed.

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Comment by xingchun-chen** [comment]
Run time delete unit s
image
image
**which version?
dev、1.3.6_prepare

**Pull request merged**